src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -1441,7 +1440,7 @@ private function registerLockConfiguration(array $config, ContainerBuilder $cont
                             $container->setDefinition($connectionDefinitionId, $connectionDefinition);
                         }
 
-                        $storeDefinition = new Definition(StoreInterface::class);
+                        $storeDefinition = new Definition(PersistStoreInterface::class);
```

Comment:
why didn't we do this in 4.4?

Reply:
done in https://github.com/symfony/symfony/pull/32558